scss/components/_video.scss

```diff
@@ -0,0 +1,38 @@
+.video-card {
+    img {
+        border-radius: calc-em(10, 16);
+    }
+    &--icon {
+        cursor: pointer;
+        right: calc-em(20, 16);
+        bottom: calc-em(20, 16);
+
+        i {
+            font-size: calc-em(36, 16);
+            color: $primary;
+        }
+
+        @include media-breakpoint-up(lg) {
+            right: calc-em(35, 16);
+            bottom: calc-em(35, 16);
+            i {
+                font-size: calc-em(54, 16);
+            }
+        }
+    }
+    @include media-breakpoint-down(md) {
+        padding-left: calc-em(15);
+        padding-right: calc-em(15);
+    }
+}
+
+.video-modal {
+    &--body{
+        padding: 0;
+    }
+    &--icon{
+        z-index: $zindex-modal;
```

Comment:
this item in the HTML can come after the video and we don't need the z-index.
please also add a space between selectors and {